tests/integration/targets/azure_ops_test_azure_manage_snapshot/tasks/create_snapshot.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Create snapshot
+  ansible.builtin.include_role:
+    name: cloud.azure_ops.azure_manage_snapshot
+  vars:
+    azure_manage_snapshot_operation: create
+    azure_manage_snapshot_name: "{{ snapshot_name }}"
+    azure_manage_snapshot_vm_name: "{{ vm_name }}"
+    azure_manage_snapshot_resource_group: "{{ resource_prefix }}-{{ resource_group }}"
```

Comment:
Better to save resource group name in a default vars
`resource_group_name: "{{ resource_prefix }}-{{ resource_group }}"`

Reply:
ack